homeassistant/components/light/httplight.py

```diff
@@ -0,0 +1,147 @@
+"""
+Support for lights that use the HTTP protocol.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.pwm/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_TYPE
+from homeassistant.components.light import (
+    Light, ATTR_BRIGHTNESS,
+    ATTR_HS_COLOR, SUPPORT_BRIGHTNESS,
+    SUPPORT_COLOR, PLATFORM_SCHEMA)
+import homeassistant.helpers.config_validation as cv
+import homeassistant.util.color as color_util
+
+
+REQUIREMENTS = ['requests==2.19.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_TYPE_SIMPLE = 'simple'
+CONF_TYPE_DIMMABLE = 'dimmable'
+CONF_TYPE_COLOR = 'color'
+
+CONF_TYPES = [CONF_TYPE_SIMPLE, CONF_TYPE_DIMMABLE, CONF_TYPE_COLOR]
+
+SUPPORT_HTTPLIGHT_BRIGHTNESS = SUPPORT_BRIGHTNESS
+SUPPORT_HTTPLIGHT_COLOR = SUPPORT_BRIGHTNESS | SUPPORT_COLOR
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
```

Comment:
Address can be anything and you don't do any error handling for your web requests